src/bin/dump/softhsm2-dump-file.cpp

```diff
@@ -461,13 +463,15 @@ void dump(FILE* stream)
 			}
 			printf("(length %lu)\n", (unsigned long) len);
 
-			std::vector<Attribute> value;
-			if (!readMap(stream, len, value))
-			{
-				corrupt(stream);
-				return;
-			}
-			dumpMap(value);
+			if (len > 0) {
```

Comment:
```suggestion
			if (len > 0)
			{
```

Reply:
Please do stylize as customary in the project, the wrong style came from my brain stem.